dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/event/RumEventSerializer.kt

```diff
@@ -64,6 +65,16 @@ internal class RumEventSerializer : Serializer<RumEvent> {
     }
 }
 
+private fun Any.toJson(): JsonElement {
+    return when (this) {
```

Comment:
is this a safety measure ? I am wondering if we still need this ...

Reply:
Well we need this because the `toJson()` method is not part of any super interface or class.